.isort.cfg

```diff
@@ -0,0 +1,6 @@
+multi_line_output=3
```

Comment:
I thought the point of https://github.com/RealOrangeOne/zoloto/pull/31/files#r305754934 was so this didn't need its own file?

Reply:
No, `pyproject.toml` isn't intended as a replacement for plugin configuration, but `isort` does parse it to help it discover things. I'm unable to find a source currently, but Guido Van Rossum was the one that made that decision